ImageResizer.AzureBlobCache/AzureBlobHandler.cs

```diff
@@ -12,6 +13,7 @@ public class AzureBlobHandler : AsyncUtils.AsyncHttpHandlerBase
         private readonly IResponseArgs syncResponse;
         private readonly IAsyncResponsePlan asyncResponse;
         private readonly Stream data;
+        private static readonly int[] ignoredHttpExceptionCodes = {-2147023667, -2147024809, -2147023901};
```

Comment:
- maybe you could add const for each number so that it's clear which code handles which situatioin?

Reply:
- if hex repesentation is more common (e.g. documentation says about hex number) maybe you could write them in code as hex: 
`uint u2 = 0x12341234; `